projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.ts

```diff
@@ -1,37 +1,69 @@
-import { AfterViewInit, Component, ElementRef, EventEmitter, HostBinding, Input,
-  OnChanges, OnDestroy, Output, SimpleChanges, ViewChild } from '@angular/core';
+import {
+  AfterViewInit, Component, ElementRef, EventEmitter, HostBinding, Input,
+  OnChanges, OnDestroy, Output, SimpleChanges, ViewChild
+} from '@angular/core';
+import { Matrix4 } from '@math.gl/core';
 import { ResizeSensor } from 'css-element-queries';
 
 /**
  * All organs that will eventually be displayed in the app
  */
 export const ALL_POSSIBLE_ORGANS: OrganInfo[] = [
-  { src: 'app:skin', organ: 'Skin', name: 'Skin', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0002097'},
-  { src: 'app:large_intestine', organ: 'Large Intestine', name: 'Large Intestine', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0000059' },
-  { src: 'app:heart', organ: 'Heart', name: 'Heart', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0000948'},
-  { src: 'app:kidney-left', organ: 'Kidney', name: 'Kidney, L', side: 'left', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0004538'},
-  { src: 'app:kidney-right', organ: 'Kidney', name: 'Kidney, R', side: 'right', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0004539'},
-  { src: 'app:spleen', organ: 'Spleen', name: 'Spleen', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0002106'},
-  { src: 'app:brain', organ: 'Brain', name: 'Allen Brain', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0000955'},
-  { src: 'app:lung', organ: 'Lung', name: 'Lungs', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0002048'},
-  { src: 'app:lymph_nodes', organ: 'Lymph Node', name: 'Lymph Node, L', side: 'left', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0000029'},
-  { src: 'app:lymph_nodes', organ: 'Lymph Node', name: 'Lymph Node, R', side: 'right', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0000029'},
-  { src: 'app:bone_marrow', organ: 'Pelvis', name: 'Pelvis', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0001270'},
-  { src: 'app:thymus', organ: 'Thymus', name: 'Thymus', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0002370'},
-  { src: 'app:vasculature', organ: 'Vasculature', name: 'Vasculature', hasSex: true,
-    id: 'http://purl.obolibrary.org/obo/UBERON_0002049'}
+  {
+    src: 'app:skin', organ: 'Skin', name: 'Skin', hasSex: true,
+    id: 'http://purl.obolibrary.org/obo/UBERON_0002097'
+  },
+  {
+    src: 'app:large_intestine', organ: 'Large Intestine', name: 'Large Intestine', hasSex: true,
+    id: 'http://purl.obolibrary.org/obo/UBERON_0000059'
+  },
+  {
+    src: 'app:heart', organ: 'Heart', name: 'Heart', hasSex: true,
+    id: 'http://purl.obolibrary.org/obo/UBERON_0000948',
+    defaultTransform: new Matrix4(Matrix4.IDENTITY).rotateX(Math.PI / 2)
```

Comment:
This needs to be [number, number, number].. Apply the matrix elsewhere, I would like to keep this 'data only'. Use the rotateXYZ function as well.